broker/src/main/java/io/zeebe/broker/bootstrap/StartProcess.java

```diff
@@ -100,11 +104,11 @@ static long takeDuration(final CheckedRunnable runner) throws Exception {
       this.startFunction = startFunction;
     }
 
-    String getName() {
+    public String getName() {
```

Comment:
Why is this necessary?

Reply:
No, this is a useless change, I'll revert this